test/UserRepository-test.js

```diff
@@ -14,7 +14,7 @@ describe('User Repository', () => {
       16,
       4,
       8
-      ]}, {
+    ]}, {
     id: 2,
     name: 'Jarvis Considine',
```

Comment:
Ensure that the info you use in tests is made up and fake (and different from the info provided). This is done for privacy/security purposes.